senza/cli.py

```diff
@@ -153,8 +153,28 @@ def convert(self, value, param, ctx):
         return key_val
 
 
+def validate_version(ctx, param, value):
+    if not VERSION_PATTERN.match(value):
+        raise click.BadParameter('Version must satisfy regular expression pattern "[a-zA-Z0-9]+"')
+    return value
+
+
+def validate_region(ctx, param, value):
+    """Validate Click region param parameter.
+    Get the list of AWS regions and check the value is in it."""
+
+    if value is not None:
+        ec2 = boto3.client('ec2', 'eu-west-1')
+        ec2_regions = ec2.describe_regions()
```

Comment:
this looks dangerously like an extra remote AWS API call....

...and it is indeed (checking the `botocore` source) :disappointed: